server/terraform/modules/myhealth/variables.tf

```diff
@@ -10,6 +10,23 @@ variable "domain" {
   type        = string
 }
 
+variable "content_bucket_id" {
+  description = "Name of Cloud Storage bucket to provision for static content."
+  type        = string
+  # Default will generate a name using the project ID.
+  default = ""
+}
```

Comment:
Definitely better if possible. That presumably means using entirely new buckets and deleting the current ones once all client apps have started using the load balancer to access them. What pattern would you like to use? It's in a flat global namespace though it can sometimes use appropriately owned DNS domain names. I'll propose something like 

who-myhealth-$projectid-static-content

Reply:
The `project_id` already includes `who-mh-xxx`, so that's all you need. Here's some example from `terraform/modules/myhealth/main.tf`:
```
  name = "${var.project_id}-lb"
  name = "${var.project_id}-url-map"
  name = "${var.project_id}-neg-appengine"
```